### DISCUSSION TITLE
MQTT Documentation

### DISCUSSION THREAD

**Discussion by HiQ-SebastianGarske** [Blackymas/NSPanel_HA_Blueprint#1273] [Q&A] [answered]
Hi,

very nice and smooth integration! I would love to use this Design with OpenHab. Is there any documentation on the MQTT from and to the panel?

**Comment by edwardtfn** [comment, marked as answer]
Thanks for your feedback.
We are using the [ESPHome native API](https://esphome.io/components/api.html), not MQTT.

**Reply by HiQ-SebastianGarske** [reply]
Thanks!
I totally missed that Point. I thought Integration was based on Tasmota over here also.

**Comment by edwardtfn** [comment]
It's probably not that hard to convert that mqqt.

**Comment by deejaybeam** [comment]
That is not the problem, but most of the logic is in the HA-Automation/Blueprint and you don´t have that with OpenHab

**Reply by edwardtfn** [reply]
True. We do a lot on Home Assistant side.
